pkg/rancher-prime/docs.ts

```diff
@@ -0,0 +1,109 @@
+    to:   'https://documentation.suse.com/cloudnative/rancher-manager/$V$/en/cluster-deployment/rke1-vs-rke2.html#_cluster_api'
+  },
+  {
+    from: 'https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-cloud-providers/amazon',
+    to:   'https://documentation.suse.com/cloudnative/rancher-manager/latest/en/cluster-deployment/set-up-cloud-providers/amazon.html'
+  },
+  {
+    from: 'https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-cloud-providers/azure',
+    to:   'https://documentation.suse.com/cloudnative/rancher-manager/latest/en/cluster-deployment/set-up-cloud-providers/azure.html'
+  },
+  {
+    from: 'https://ranchermanager.docs.rancher.com/reference-guides/cluster-configuration/rancher-server-configuration/rke2-cluster-configuration#additionalmanifest',
+    to:   'https://documentation.suse.com/cloudnative/rancher-manager/latest/en/cluster-deployment/configuration/k3s.html#_additionalmanifest',
+  },
+  {
+    from: 'https://ranchermanager.docs.rancher.com/$V$/integrations-in-rancher/logging/logging-helm-chart-options',
+    to:   'https://documentation.suse.com/cloudnative/rancher-manager/$V$/en/observability/logging/logging-helm-chart-options.html'
+  },
+  {
+    from: 'https://ranchermanager.docs.rancher.com/$V$/monitoring-alerting/',
+    to:   'https://documentation.suse.com/cloudnative/rancher-manager/$V$/en/observability/monitoring-and-dashboards/monitoring-and-dashboards.html',
+  },
+  {
+    from: 'https://ranchermanager.docs.rancher.com/$V$/faq/telemetry',
+    to:   'https://documentation.suse.com/cloudnative/rancher-manager/$V$/en/faq/telemetry.html'
+  },
+  {
+    from: 'https://ranchermanager.docs.rancher.com/$V$/reference-guides/cluster-configuration/rancher-server-configuration/aks-cluster-configuration#support-private-kubernetes-service',
+    to:   'https://documentation.suse.com/cloudnative/rancher-manager/$V$/en/cluster-deployment/hosted-kubernetes/aks/configuration.html#_support_private_kubernetes_service'
+  },
+  {
+    from: 'https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/gke',
+    to:   'https://documentation.suse.com/cloudnative/rancher-manager/latest/en/cluster-deployment/hosted-kubernetes/gke/gke.html'
+  },
+  {
+    from: 'https://docs.rke2.io/install/containerd_registry_configuration',
+    to:   'https://documentation.suse.com/cloudnative/rke2/latest/en/install/containerd_registry_configuration.html'
+  },
+  {
+    from: 'https://docs.rke2.io/security/hardening_guide',
+    to:   'https://documentation.suse.com/cloudnative/rke2/latest/en/security/hardening_guide.html'
+  },
+  {
+    from: 'https://docs.k3s.io/installation/private-registry',
+    to:   'https://documentation.suse.com/cloudnative/k3s/latest/en/installation/private-registry.html'
```

Comment:
does `https://docs.rke2.io/install/private_registry` also need doing?

Reply:
Added - this link was not provided by docs, but I've added what I think is the correct link